scripts/generate-board.py

```
@@ -0,0 +1,253 @@
+        "cards_under_odds":under_map(od,CARD_UNDER,"Cards"),"btts_no_odds":od.get("BTTS No"),
+        "value":{"goals":{mk:value_cell(goals_over[mk],goals_odds[mk]) for mk in GOAL_MARKS},
+                 "corners":{mk:value_cell(corners_over[mk],corners_odds[mk]) for mk in CORNER_MARKS},
+                 "btts":value_cell(btts_pct,od.get("BTTS"))},
+        "home_form":games_for(f["homeId"],f["homeName"])[:8],"away_form":games_for(f["awayId"],f["awayName"])[:8],
+        "h2h":h2h_for(f["homeId"],f["homeName"],f["awayId"],f["awayName"])[:6]}
+    fixtures.append(row)
+
+# ── merge: keep recent PAST days (frozen as locked), refresh today→+2 ────────
+existing=[]
+try:
+    with open(OUT) as fp: existing=json.load(fp).get("fixtures",[])
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)